backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -254,9 +264,16 @@ def create_user(self, context, request: TracimRequest, hapic_data=None):
             config=app_config,
         )
         groups = [gapi.get_one_with_name(hapic_data.body.profile)]
+        # FIXME - G.M - 2018-11-27 - do not allow send email
+        # notification without valid password
```

Comment:
There is an issue related to this FIXME ?

Reply:
already fix in https://github.com/tracim/tracim/pull/1201